zds/middlewares/ForceHttpsMembersMiddleware.py

```diff
@@ -0,0 +1,19 @@
+from django.conf import settings
+from django.core.urlresolvers import reverse
+from django.http import HttpResponseRedirect
+
+
+class ForceHttpsMembersMiddleware(object):
+    """
+    Force the https when the member is logged or if it's the log in/sign up page
+    """
+
+    def process_request(self, request):
+        if getattr(settings, 'FORCE_HTTPS_FOR_MEMBERS', False) and not request.is_secure():
+            if request.user.is_authenticated() or \
+                    request.path == reverse('zds.member.views.login_view') or \
```

Comment:
perso, je ferai un attribut (voir un setting) `forced_https_urls` qui vaudrait 

```python
forced_https_urls = [
    reverse('zds.member.views.login_view'),
    reverse('register-member')
    ]
```

puis je mettrais `request.path in forced_https_urls` afin qu'on puisse ajouter ou retirer très facilement des urls de cette liste.